pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -17,6 +17,7 @@ import (
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	genericmux "k8s.io/apiserver/pkg/server/mux"
 	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/rest"
 	restclient "k8s.io/client-go/rest"
```

Comment:
these are the same : ) noting here so I don't forget to fix

Reply:
Whoops